mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go

```diff
@@ -1903,6 +1903,19 @@ type TableReference struct {
 }
 
 func resourceBigQueryTableUpdate(d *schema.ResourceData, meta interface{}) error {
+	// If only client-side fields were modified, short-circuit the Update function to avoid sending an update API request.
+	clientSideFields := map[string]bool{"deletion_protection": true}
+	clientSideOnly := true
+	for field := range ResourceBigQueryTable().Schema {
+		if d.HasChange(field) && !clientSideFields[field] {
```

Comment:
I assume nested changes will be caught by `d.HasChange(field)`. Would be possible adding a test only changing a nested subfield?

Reply:
Sure, added.